migrations/violet_background.js

```diff
@@ -0,0 +1,35 @@
+var migrationName = 'violet_backgrounds.js';
+
+/*
+ * This migration will find users with unlimited subscriptions who are also eligible for Jackalope mounts, and award them
+ */
+import Bluebird from 'bluebird';
+
+import { model as User } from '../website/server/models/user';
+
+async function addVioletBGs () {
+  let userCount = 0;
+  let promises = [];
+  let cursor = User.find({
+    'purchased.background.violet': {$ne: true},
```

Comment:
I think this search will be very slow because there's no index for it. Is a query needed here? If the script has to be run more than once, it won't hurt to reassign the background to any users who have it (apart from the time take to assign it unnecessarily, but that might be less than from running a slow query). Or you could query for the migration name, as long as no other migrations are run until you're certain all users have the background.